packages/flutter/lib/src/widgets/app.dart

```diff
@@ -235,9 +235,8 @@ class WidgetsApp extends StatefulWidget {
   /// A one-line description used by the device to identify the app for the user.
   ///
   /// On Android the titles appear above the task manager's app snapshots which are
-  /// displayed when the user presses the "recent apps" button. Similarly, on
-  /// iOS the titles appear in the App Switcher when the user double presses the
-  /// home button.
+  /// displayed when the user presses the "recent apps" button. On iOS this
+  /// value cannot be used. The app's `Info.plist` is referred to instead.
```

Comment:
Maybe mention the key in the Info.plist that is read. I think it is the `CFBundleProductName`

Reply:
Updated. From what I can tell it's normally [`CFBundleDisplayName`](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/TP40009249-110725-TPXREF135), but falls back to [`CFBundleName`](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/TP40009249-109585) when that's unset.